Packs/CheckPointHEC/Integrations/CheckPointHEC/CheckPointHEC.py

```diff
@@ -384,6 +656,22 @@ def fetch_incidents(client: Client, first_fetch: str, saas_apps: List[str], stat
 
     counter = 0
     incidents: list[dict[str, Any]] = []
+
+    if collect_restore_requests:
```

Comment:
Can you please explain what do you mean with restore_request? some requests are being restored?

Reply:
When an email is detected as a phishing or spam and the policies move it to quarantine or spam folder, the email user can report a mis-classification, we call this a restore request. Our customers asked us to include restore requests as incidents, so we implemented a new boolean parameter in the configuration, if it is enabled, then we retrieve restore requests as incidents.

Reply:
Great! thank you for the explaination